src/views/Results.vue

```diff
@@ -380,16 +394,22 @@ export default class Results extends ResultsProps {
     // Close the dialog
     this.dialog = false;
 
-    // If just one file, focus it
-    if (ids.length === 1) {
-      this.$router.push(`/results/${ids[0]}`);
+    //enable all uploaded files
+    let filter_module = getModule(FilteredDataModule, this.$store);
+    for (let i of ids) {
+      filter_module.set_toggle_file_on(i);
     }
+  }
 
-    // If more than one, focus all.
-    // TODO: Provide support for focusing a subset of files
-    else if (ids.length > 1) {
-      this.$router.push(`/results/all`);
+  get info_width(): number {
+    let mod = getModule(ServerModule, this.$store);
+    if (mod.serverMode == undefined) {
+      mod.server_mode();
+    }
```

Comment:
There has to be some way to load whether or not the application is in server mode on startup. Perhaps using on app `mounted` or something so that this check doesn't have to be done every time, this PR alone has 5 instances of this block of code in it.

Reply:
Nevermind, this is refactored in #277, I'll have to resolve this in my PR when this is merged.